internal/cli/cli.go

```diff
@@ -138,17 +138,10 @@ func (c *cli) setup(ctx context.Context) error {
 		ua = fmt.Sprintf("%v/%v", userAgent, strings.TrimPrefix(buildinfo.Version, "v"))
 	)
 
-	if t.ClientID != "" && t.ClientSecret != "" {
-		m, err = management.New(t.Domain,
-			management.WithClientCredentials(t.ClientID, t.ClientSecret),
-			management.WithUserAgent(ua),
-		)
-	} else {
-		m, err = management.New(t.Domain,
-			management.WithStaticToken(t.AccessToken),
-			management.WithUserAgent(ua),
-		)
-	}
+	m, err = management.New(t.Domain,
```

Comment:
This is one of the major simplifications that occurs when we rely simply on the access token for both authentication methods.